package-lock.json

```diff
@@ -3087,24 +3084,11 @@
 			"resolved": "https://registry.npmjs.org/yallist/-/yallist-3.1.1.tgz",
 			"integrity": "sha512-a4UGQaWPH59mOXUYnAG2ewncQS4i4F43Tv3JoAM+s2VDAmS9NsK8GpDMLrCHPksFT7h3K6TOoUNn2pb7RoXx4g=="
 		},
-		"node_modules/@sentry/browser": {
-			"version": "5.30.0",
-			"resolved": "https://registry.npmjs.org/@sentry/browser/-/browser-5.30.0.tgz",
-			"integrity": "sha512-rOb58ZNVJWh1VuMuBG1mL9r54nZqKeaIlwSlvzJfc89vyfd7n6tQ1UXMN383QBz/MS5H5z44Hy5eE+7pCrYAfw==",
-			"dependencies": {
-				"@sentry/core": "5.30.0",
-				"@sentry/types": "5.30.0",
-				"@sentry/utils": "5.30.0",
-				"tslib": "^1.9.3"
-			},
-			"engines": {
-				"node": ">=6"
-			}
-		},
 		"node_modules/@sentry/core": {
```

Comment:
strange, why is sentry/core not removed as well?

Reply:
Yes, it's not removed but go down to the dev dependencies.
It's add by [sass-lint-auto-fix](https://www.npmjs.com/package/sass-lint-auto-fix)